src/target/generated.rs

```diff
@@ -1516,6 +1516,18 @@ pub(crate) const LIST: &[(&str, TargetInfo<'static>)] = &[
             unversioned_llvm_target: "mips-unknown-linux-uclibc",
         },
     ),
+    (
+        "mips-zkm-zkvm-elf",
```

Comment:
Thank you!


Please do not edit this file directly

This file is generated by https://github.com/rust-lang/cc-rs/tree/main/dev-tools/gen-target-info/src

So when new targets are added in rustc, this file should be automatically regenerated in CI.